crafters/nlp/PDFExtractorSegmenter/tests/test_pdfextractorcrafter.py

```diff
@@ -0,0 +1,118 @@
+from .. import PDFExtractorSegmenter
+from PIL import Image
+import os
+
+expected_text = "A cat poem\nI love cats, I love every kind of cat,\nI just wanna hug all of them, but I can't," \
+                "\nI'm thinking about cats again\nI think about how cute they are\nAnd their whiskers and their " \
+                "nose\n"
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+path1 = os.path.join(cur_dir, 'cats_are_awesome.pdf')
+path2 = os.path.join(cur_dir, 'cats_are_awesome_text.pdf')
+path3 = os.path.join(cur_dir, 'cats_are_awesome_img.pdf')
```

Comment:
a better naming is preferred.
```suggestion
path_img_text = os.path.join(cur_dir, 'cats_are_awesome.pdf')
path_text = os.path.join(cur_dir, 'cats_are_awesome_text.pdf')
path_img = os.path.join(cur_dir, 'cats_are_awesome_img.pdf')
```